sopel/modules/remind.py

```diff
@@ -503,3 +503,62 @@ def remind_at(bot, trigger):
         bot.reply('Okay, will remind at %s' % human_time)
     else:
         bot.reply('Okay, will remind in %s secs' % duration)
+
+
+@plugin.command('reminders')
+@plugin.example('.reminders forget *', user_help=True)
+@plugin.example('.reminders count #channel', user_help=True)
+@plugin.example('.reminders count', user_help=True)
+def manage_reminders(bot, trigger):
+    """Count or forget your reminders in the current channel.
+
+    Use a subcommand "count" (default) or "forget". The second argument is
+    optional and can be either a channel name, your nick, or * (for all).
+    """
+    owner = trigger.nick
+    action = trigger.group(3) or 'count'
+    target = trigger.group(4) or trigger.sender
```

Comment:
I know you were trying not to make it _too_ smart (and therefore complicated), but as an alternative to complaining at the user if they forget to specify `count` and just put a context:

```suggestion
    action = trigger.group(3)
    target = trigger.group(4) or trigger.sender
    
    if action not in ['trigger', 'count']:
        # forgetful `.reminders mynick` usage; treat as `.reminders count mynick`
        target = action
        action = 'count'
```

Reply:
I'll add a `and not target` and remove the default for target, because there is no reason to accept `.reminders wrong-action #channel`, but `.reminders #channel` is fine by me.